framework/widgets/ActiveField.php

```diff
@@ -315,10 +317,17 @@ public function error($options = [])
      *
      * @return $this the field object itself
      */
-    public function hint($content, $options = [])
+    public function hint($content = null, $options = [])
```

Comment:
What is the problem with this particular change? It brings the method signature in line with `label()` method and does not break BC in any way.